.github/workflows/test-pr.yml

```
@@ -77,7 +77,7 @@ jobs:
           sudo apt-get -y install graphviz ./kframework_amd64_ubuntu_jammy.deb
           kompile --version
       - name: 'Build and run integration tests'
-        run: make cov-integration
+        run: make cov-integration TEST_ARGS='-n8'
```

```suggestion
        run: make cov-integration COVER_ARGS='-n8'
```

Does this make execution faster, i.e. do `normal-ephemeral` instances have enough cores?

---

Yes, they should have 12 cores each.